assets/javascripts/templates/connectors/discovery-list-container-top/category-events.hbs

```diff
@@ -0,0 +1 @@
+<div id="upcoming-events-calendar"></div>
```

Comment:
Could we use the existing `upcoming-events-calendar` component here? It feels like we would be reusing some code/logic and just need to change the model.

Reply:
I tried but the issue here is I am not sure how to set the model from inside the initializer condition.

https://github.com/discourse/discourse-calendar/blob/aff34c012a51f9d6c946de6c3741eebfb5a57a68/assets/javascripts/discourse/templates/discourse-post-event-upcoming-events-index.hbs#L2

Reply:
Ah, yes, in the connector it's a little trickier to set this up. It's probably fine as is then.